plugins/infinispan60/src/main/java/org/radargun/service/AbstractInfinispanQueryable.java

```diff
@@ -141,22 +141,37 @@ private FilterConditionEndContext getEndContext(String attribute) {
       }
 
       @Override
-      public Query.Builder orderBy(String attribute, Query.SortOrder order) {
+      public Query.Builder orderBy(Query.SelectExpression selectExpression, Query.SortOrder order) {
          if (builder == null) throw new IllegalArgumentException("You have to call orderBy() on root query builder!");
-         builder.orderBy(attribute, order == Query.SortOrder.ASCENDING ?
+         if (selectExpression.function != Query.AggregationFunction.NONE) throw new IllegalArgumentException("This version of infinispan doesn't support aggregations!");
```

Comment:
Which version of Infinispan added aggregations? I'm just wondering why this code isn't in the correct version of the plugin to avoid this possible failure?